test/bdd/fixtures/universal-registrar/docker-compose.yml

```diff
@@ -4,28 +4,25 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-version: "3.5"
-networks:
-  default:
-    name: universal-registrar
+version: '3'
+
 services:
-  driver-did-sov:
-    image: universalregistrar/driver-did-sov:latest
-    environment:
-      uniregistrar_driver_did_sov_libIndyPath: ${UNIREGISTRAR_DRIVER_DID_SOV_LIBINDYPATH}
-      uniregistrar_driver_did_sov_poolConfigs: ${UNIREGISTRAR_DRIVER_DID_SOV_POOLCONFIGS}
-      uniregistrar_driver_did_sov_poolVersions: ${UNIREGISTRAR_DRIVER_DID_SOV_POOLVERSIONS}
-      uniregistrar_driver_did_sov_walletName: ${UNIREGISTRAR_DRIVER_DID_SOV_WALLETNAME}
-      uniregistrar_driver_did_sov_trustAnchorSeed: ${UNIREGISTRAR_DRIVER_DID_SOV_TRUSTANCHORSEED}
-    ports:
-      - "9082:9080"
   driver-did-v1:
     image: universalregistrar/driver-did-v1:latest
-    environment:
-      uniregistrar_driver_did_v1_trustAnchorSeed: ${UNIREGISTRAR_DRIVER_DID_V1_TRUSTANCHORSEED}
```

Comment:
the seed isn’t needed?

Reply:
not needed for driver-did-v1 https://github.com/decentralized-identity/universal-registrar/blob/3c26c4c1abc338ff502ea8eedeedd4538e713836/.env#L18